packages/ui/src/components/TanstackTable.tsx

```diff
@@ -604,60 +586,53 @@ export function TanstackTableCard<RowDataModel>({
           </div>
         </div>
       </div>
-      <div class="card-body d-flex flex-column">
-        <div class="d-flex flex-row flex-wrap align-items-center mb-3 gap-2">
-          <div class="flex-grow-1 flex-lg-grow-0 col-xl-6 col-lg-7 d-flex flex-row gap-2">
-            <div class="input-group">
-              <input
-                ref={searchInputRef}
-                type="text"
-                class="form-control"
-                aria-label={globalFilter.placeholder}
-                placeholder={globalFilter.placeholder}
-                value={globalFilter.value}
-                onInput={(e) => {
-                  if (!(e.target instanceof HTMLInputElement)) return;
-                  globalFilter.setValue(e.target.value);
-                }}
-              />
-              <button
-                type="button"
-                class="btn btn-outline-secondary"
-                aria-label="Clear search"
-                title="Clear search"
-                data-bs-toggle="tooltip"
-                onClick={() => globalFilter.setValue('')}
-              >
-                <i class="bi bi-x-circle" aria-hidden="true" />
-              </button>
-            </div>
-            {/* We do this instead of CSS properties for the accessibility checker.
-              We can't have two elements with the same id of 'column-manager-button'. */}
-            {isMediumOrLarger && (
-              <>
-                <ColumnManager table={table} />
-                {columnManagerButtons}
-              </>
+      <div class="card-body d-flex flex-row flex-wrap flex-grow-0 align-items-center gap-2">
```

Comment:
I adjusted the markup here so that the table could span the full width of the card and extend all the way to the bottom.